crates/re_viewer/src/ui/data_blueprint.rs

```diff
@@ -71,6 +71,20 @@ pub struct DataBlueprintTree {
     data_blueprints: DataBlueprints,
 }
 
+// Manually implement PartialEq since slotmap doesn't
+impl PartialEq for DataBlueprintTree {
+    fn eq(&self, other: &Self) -> bool {
+        self.groups
```

Comment:
You should use the exhaustive `let Self { groups, path_to_group, … } = self` destruction trick here to make sure this gets updated when `DataBlueprintTree` changes